src/cloud-api-adaptor/ibmcloud-powervs/README.md

```diff
@@ -30,27 +30,33 @@ pvsadm image qcow2ova --prep-template-default > image-prep.template
 
 Add the following snippet to `image-prep.template`
 ```
-yum install -y gcc gcc-c++ git make
+yum install -y gcc gcc-c++ git make wget perl
+wget https://github.com/mikefarah/yq/releases/download/v4.42.1/yq_linux_ppc64le
+chmod +x yq_linux_ppc64le && mv yq_linux_ppc64le /usr/local/bin/yq
 git clone -b main https://github.com/kata-containers/kata-containers.git
```

Comment:
do we need to clone the repo - as it is being taken care of [here](https://github.com/confidential-containers/cloud-api-adaptor/blame/db56a2d3b30ff7e42cc100e6616ef049cdcc03b8/src/cloud-api-adaptor/podvm/Makefile.inc#L140)?

Reply:
Will remove it, as was picked from the recent changes